### PR TITLE
Added support for base 10

### PR DESCRIPTION
**Pull request by moul**
Inspired by the opposite module here: https://github.com/avoidwork/filesize.js/blob/master/lib/filesize.js

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,3 +1,4 @@
+sudo: false
```
Comment (patrickkettner) [inline .travis.yml]: thanks for adding that!

File: README.md [hunk 2]
```diff
@@ -14,3 +15,18 @@ filesizeParser('200KB')
 filesizeParser('200Kb')
 25600
 ```
+
+The default [base](https://en.wikipedia.org/wiki/Gigabyte) is `2`, you can use base `10`
```
Comment (patrickkettner) [inline README.md]: Could you update to point to [the base section](https://en.wikipedia.org/wiki/Gigabyte#Base_2_definition)?

**Comment by patrickkettner** [comment]
beautiful work! couple small nits, then lgtm!

File: README.md (hunk 2 above)
Comment (moul) [inline README.md]: Done and rebased

**Comment by patrickkettner** [comment]
you are wonderful

**Comment by moul** [comment]
:ok_hand:

**Pull request merged**